specification/azurelargeinstance/resource-manager/readme.md

```diff
@@ -0,0 +1,91 @@
+# azurelargeinstance
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for azurelargeinstance.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the azurelargeinstance.
+
+```yaml
+openapi-type: arm
+tag: package-2023-07-20-preview
+```
+
+### Tag: package-2023-07-20-preview
+
+These settings apply only when `--tag=package-2023-07-20-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-07-20-preview'
+input-file:
+  - Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
+  - repo: azure-resource-manager-schemas
+  - repo: azure-cli-extensions
+  - repo: azure-powershell
+```
+## Az
+
+See configuration in [readme.az.md](./readme.az.md)
+
+## Go
+
+See configuration in [readme.go.md](./readme.go.md)
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
+## TypeScript
+
+See configuration in [readme.typescript.md](./readme.typescript.md)
+
+## CSharp
+
+See configuration in [readme.csharp.md](./readme.csharp.md)
+
+### Suppress linting rules
+
+These set of linting rules aren't applicable to the AzureLargeInstance RP so suppressing them here.
+
+``` yaml
+suppressions:
+  - code: TrackedResourcesMustHavePut
+  - reason: All PUT actions are carried out internally by our specialized team utilizing Geneva actions. This process is not currently available for external use.
+  - where:
```

Comment:
Hi @8Gitbrix , please remove the '-' in these two places.

Reply:
Thanks. @Alancere, I'm still having issues with the format of the suppression, do you know what I'm doing wrong?